modules/conversantAnalyticsAdapter.js

```diff
@@ -529,6 +644,10 @@ conversantAnalytics.enableAnalytics = function (config) {
   // Math.random() pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1)
   cnvrHelper.doSample = Math.random() < initOptions.cnvr_sample_rate;
 
+  if (initOptions.send_error_data !== undefined && initOptions.send_error_data !== null && typeof initOptions.send_error_data === 'boolean') {
```

Comment:
There is a `isBoolean` function in utils.js.  However, it's usually sufficient to do a double-negation like this:

```javascript
cnvrHelper.doSendErrorData = !!initOptions.send_error_data;
```

Reply:
a bit obscure but I added a comment to inform of what it's doing